policies/wasm/policies/namespace_discovery_relationship_policy/namespace_discovery_relationship_policy.rego

```diff
@@ -1,62 +1,54 @@
 # https://play.openpolicyagent.org/p/wFNhyGsIej
 package hierarchical_policy
 
-available_namespaces[namespace_key] {
-  service := input.services[_];
-  namespace_key = service.namespace
+import rego.v1
+
+available_namespaces[namespace_key] if {
+	some service in input.services
+	namespace_key := service.namespace
 }
 
-available_namespaces[namespace_key] {
-  service := input.services[_];
-  service.type == "Namespace";
-  namespace_key = service.name
+available_namespaces[namespace_key] if {
+	some service in input.services
+	service.type == "Namespace"
+	namespace_key := service.name
 }
 
-parent_child_mapping = { namespace_map: ns |
-	some namespace_map
-	available_namespaces[namespace_map]
-    service := input.services[key]
-    service.namespace
-    service.namespace == namespace_map
-    ns := {ns_serv: comp_id |
-        some ns_serv
-         x := input.services[ns_serv]
-         x.namespace == namespace_map
-         comp_id := x.traits.meshmap.id
-    }
+parent_child_mapping[namespace_map] := ns if {
+	some namespace_map in available_namespaces
+	some key
+	service := input.services[key]
```

Comment:
This is fine, but:
```rego
some key
service := input.services[key]
```
could just as well be rewritten as:
```rego
some key, service in input.services
```
and perhaps it'd be more consistent